lib/config.js

```diff
@@ -279,6 +279,7 @@ class Config {
           globalLabels: maybePairsToObject(conf.globalLabels),
           hostname: conf.hostname,
           environment: conf.environment,
+          metadataFilters: agent._metadataFilters,
```

Comment:
I don't see where/how this is used.  There's no `agent._metadataFilters` property assigned anywhere else -- `addMetadataFilter` adds its functions directly to the transport/client object.  Old code, or something else?

Reply:
Good catch. Cruft left over from an earlier impl. Dropped in commit 26b897a.